backend/django/core/templates/projects/update/ml.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% block page_title %}Update Project{% endblock %}
+{% load static %}
+{% load render_bundle from webpack_loader %}
+
+{% block content %}
+<div class="overlay">
+    <div class="overlay_contents">
+        <div id="second_progress"></div>
+        <span id="loading_text"></span>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-8 col-md-offset-2">
+        <div class="card">
+            <div class="cardface">
+                {{ form.media.css }}
+                <form action="." method="post" enctype="multipart/form-data" onsubmit="addASpinner()">
+                {% csrf_token %}
+                <h1>ML Settings</h1>
+                <h3>Description</h3>
+                <p>By default, SMART uses a trained machine-learning model to find the best label suggestions for data.</p>
+                <p>If you would like to add weighted text pairs to further train the SMART model for this project, upload a .csv file:</p>
+                <h3>Instructions</h3>
+                <p>Upload a .csv file with weighted text pairs:</p>
+                <p><a href="{% static 'example_ml.csv' %}">An example pairs file can be downloaded from here</a>.</p>
```

Comment:
What do the 0.7 and 0.3 in this example file mean? I think we need to add some more information to help SME's to know how to set up these files.